### ISSUE TITLE
unexpected error during slicing

### ISSUE THREAD

**Issue by pflonsch** [Ultimaker/Cura#12611]
### Application Version

5.0.0

### Platform

Windows 10

### Printer

Creality Ender-3 Pro

### Reproduction steps

1. Loaded file "LVL2_LibraryRightA.stp"
2. Clicked "Slice" with print settings I used several times before
3. Tried to rotate object
4. Tried to restart Cura
![Slicing-Error](https://user-images.githubusercontent.com/108240903/175834906-eccc2211-2893-4a73-8515-387f50322d2d.PNG)

### Actual results

Cura stopped slicing with an Error message.

### Expected results

Cura should slice, or give an error message with a hint to what could be the problem.

### Checklist of files to include

- [X] Log file
- [X] Project file

### Additional information & file uploads

[LVL2_LibraryRightA.zip](https://github.com/Ultimaker/Cura/files/8987575/LVL2_LibraryRightA.zip)
[cura.log](https://github.com/Ultimaker/Cura/files/8987577/cura.log)

**Comment by GregValiant** [comment]
I made no changes to your 3mf file and it sliced fine for me.  Then I moved it on the build plate and it wouldn't slice.
There have been a number of reports lately on this sort of behavior with 5.0 and it's been for different reasons.  This does not appear to be a model problem.  Sometimes it slices and sometimes it hangs.  As a workaround - try moving it here and there on the build plate, change the "Initial Layer Height" up or down by .01mm, rotate it a little bit, that sort of thing.
This was the first attempt.  I was feeling pretty good there for about a minute.
![image](https://user-images.githubusercontent.com/64202104/175841236-50348758-08f9-4d5a-95f3-ea14d5649002.png)
